openquake/mbt/tools/mfd.py

```diff
@@ -182,7 +182,7 @@ def get_moment_from_mfd(mfd, threshold=-1):
         mo_tot = 0.0
         for occ in occ_list:
             if occ[0] > threshold:
-                mo_tot += occ[1] * 10.**(1.5*occ[0] + 9.05)
+                mo_tot += occ[1] * 10.**(1.5*occ[0] + 9.1)
```

Comment:
I would prefer keeping 9.05 for consistency with other functions e.g.  openquake/hazardlib/mfd/truncated_gr.py. See also discussion at page 560 in https://pubs.geoscienceworld.org/ssa/bssa/article/89/3/559/120382/a-mutually-consistent-seismic-hazard-source-model

Reply:
OK, then I would also change the rest of the functions that use 9.1 (`mfd.mag_to_mo` and `mfd.mo.to.mag`, `faults.get_rate_above_m_low`, etc.) so that it's consistent.